src/nxdoc/getting-started/tracks/package-publish-deploy-hotreload.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 4 - Package \ Publish \ Deploy \ Hotreload
+review:
+    comment: ''
+    date: '2021-03-02'
+    status: ok
+details:
+    howto:
+        excerpt: Learn how to package, publish, deploy and hotreload a project.
+        level: Intermediate
+        tool: 'Nuxeo CLI, Nuxeo Studio'
+        topics: 'Automation, Layout'
+labels:
+    - howto
+    - nuxeo-cli
+toc: true
+tree_item_index: 300
+history:
+    -
+        author: Florian Bématol
+        date: '2021-03-02 15:26'
+        message: ''
+        version: '1'  
+---
+
+{{#> callout type='info'}}
+Watch the related courses on Nuxeo University
+- [Course on Nuxeo CLI](https://university.nuxeo.com/learn/public/course/view/elearning/83/NuxeoDevTools)
+{{/callout}}
+
+## Bootstrap your nuxeo package
+
+1.  In your terminal:
+
+    ```bash
+    $ nuxeo bootstrap package
+    ```
+
+2.  Link your project:
+
+    ```bash
+    $ nuxeo studio link
+    ```
+
+## Create project your docker image
+
+1.  In your terminal:
+
+    ```bash
+    $ nuxeo bootstrap docker-compose
+    ```
+
+## Run your project
+
+1.  In your terminal:
+
+    ```bash
+    $ docker-compose up [-d]
+    ```
+
+2.  If you want to follow the logs:
+
+    ```bash
+    $ docker-compose logs -f
+    ```
+
+#### Hotreload
+
+## Hotreload - Navigateur
+
+1.  Install extension
+
+2.  On your browser open your nuxeo application
+
+3.  Click extension logo, click hotreload button.
+
+## Hotreload - NuxeoCLI
+
+1.  Optional, you can configure nuxeoctl hotreload with:
+
+    1.  In your terminal:
+
+    ```bash
+    $ nuxeo hotreload configure
+    ```
+
+    2.  Fill up questions:
+    
+    ```bash
+    ? Nuxeo Server path: ??
+    ??..
+    ```
+
+2.  In your terminale:
+
+    ```bash
+    $ nuxeo hotreload
+    ```
+
+## Release your project
+
+1.  Go to **Studio** | Access studio project page.
+
+2.  Under **Source Control** menu | Go to **Branch Management**.
+
+3.  Select your current branch.
+
+4.  Click release button.
+
+5.  Update your pom.xml with the proper Studio version
+
+6.  Release your project using maven command:
+
+    ```bash
+    $ mvn release 
```

Comment:
we have to retest this one, I guess it's more like `mvn version:release + some args`